http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBootSpec.scala

```diff
@@ -242,6 +335,22 @@ class MonitorAtBootSpec extends AnyFlatSpec with TestComponent with LeonardoTest
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "ignore non-monitored apps in Azure" in isolatedDbTest {
```

Comment:
what's non-monitored apps?

Reply:
any app that doesnt have the status Prvisioning or Deleting when leo starts up